src/org/pentaho/reporting/platform/plugin/async/AsyncReportStatusListener.java

```diff
@@ -28,7 +28,7 @@
  * Simple synchronized bean with async execution status. This class is part of the PentahoAsyncReportExecution
  * implementation and should not be used outside of that.
  */
-class AsyncReportStatusListener implements IAsyncReportListener {
+public class AsyncReportStatusListener implements IAsyncReportListener {
 
```

Comment:
The AsyncReportStatusListener is an implementation details of the AsyncReportExecution and is not useful outside of that context. There is no need to make that class public.